src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
@AnshulMalik  good but it is not related to #100 issue.if we can do only related changes in commits.that will be really easy to others.